utility/module.go

```diff
@@ -0,0 +1,46 @@
+package utility
+
+import (
+	"log"
+	"pocket/shared/config"
+	"pocket/shared/modules"
+)
+
+var _ modules.UtilityModule = &utilityModule{}
+
+type utilityModule struct {
+	pocketBus modules.Bus
+}
+
+func Create(cfg *config.Config) (modules.UtilityModule, error) {
```

Comment:
I'd prefer if you made this param explicitly not used

`func Create(_ *config.Config) (modules.UtilityModule, error) {`

Reply:
Done. Updated persistence module as well.